pkg/sentry/identity/kubernetes/validator_test.go

```diff
@@ -0,0 +1,43 @@
+package kubernetes
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	fake "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestValidate(t *testing.T) {
+	t.Run("invalid token", func(t *testing.T) {
+		fakeClient := fake.NewSimpleClientset()
```

Comment:
NewSimpleClientSet returns fake which always returns happy-path client. 

1.First we need to use empty fake client
https://github.com/dapr/dapr/blob/e540a7166aeaf115773ccc4c7a1056ae7eed073b/tests/platforms/kubernetes/appmanager_test.go#L42

2. Set reactor for CreateVerb 
https://github.com/dapr/dapr/blob/e540a7166aeaf115773ccc4c7a1056ae7eed073b/tests/platforms/kubernetes/appmanager_test.go#L88-L114

I found that this is the common way to write unit-test using Reactor when we contributes to k8s. So I use this way to validate e2e test framework.